Commander/ArgumentDescription.swift

```diff
@@ -122,35 +122,52 @@ public class Options<T : ArgumentConvertible> : ArgumentDescriptor {
 
 public class Flag : ArgumentDescriptor {
   public typealias ValueType = Bool
-
+  
   public let name:String
+  public let falseName:String?
   public let flag:Character?
+  public let falseFlag:Character?
   public let description:String?
   public let `default`:ValueType
   public var type:ArgumentType { return .Option }
-
-  public init(_ name:String, flag:Character? = nil, description:String? = nil, `default`:Bool = false) {
+  
+  public init(_ name:String, falseName:String? = nil, flag:Character? = nil, falseFlag:Character? = nil, description:String? = nil, `default`:Bool = false) {
```

Comment:
After thinking about this for a while, I'd rather call this `disabledName` and `disabledFlag`. Would you be able to change the name to this?

Can you keep the old order intact so that the flag is after the original name, as follows:

```swift
init(_ name: String, flag: Character? = nil, disabledName: String? = nil, disabledFlag: Character? = nil, description: String? = nil, `default`: Bool = false)
```